app/src/main/java/com/amaze/filemanager/asynchronous/services/CopyService.java

```diff
@@ -85,7 +85,9 @@
 
     private long totalSize = 0L;
     private int totalSourceFiles = 0;
+    private String notificationID;
```

Comment:
Why String if you are going to parse it to int every time?

Reply:
Too much time consuming to rewrite everything. Will do later.

Reply:
#876 has cleaned most of it, will the string through...